cli/limit.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"sort"
+	"strings"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/controller/client"
+	ct "github.com/flynn/flynn/controller/types"
+	"github.com/flynn/flynn/host/resource"
+	"github.com/flynn/flynn/pkg/typeconv"
+)
+
+func init() {
+	register("limit", runLimit, `
+usage: flynn limit [-t <proc>]
+       flynn limit set <proc> <var>=<val>...
+
+Manage app resource limits.
+
+Options:
+	-t, --process-type=<proc>  set or read limits for specified process type
+
+Commands:
+	With no arguments, shows a list of resource limits.
+
+	set    sets value of one or more resource limits
+
+Examples:
+
+	$ flynn limit
+	web:     max_fd=10000  max_procs=256  memory=1GB
+	worker:  max_fd=10000  max_procs=256  memory=1GB
+
+	$ flynn limit set web memory=512MB max_procs=1024 max_fd=12000
+	Created release 5058ae7964f74c399a240bdd6e7d1bcb
+
+	$ flynn limit
+	web:     max_fd=12000  max_procs=1024  memory=512MB
+	worker:  max_fd=10000  max_procs=256   memory=1GB
+
+	$ flynn limit set web memory=256MB
+	Created release b39fe25d0ea344b6b2af5cf4d6542a80
+
+	$ flynn limit
+	web:     max_fd=12000  max_procs=1024  memory=256MB
+	worker:  max_fd=10000  max_procs=256   memory=1GB
```

Comment:
We need to drop max_procs, as it doesn't work (and won't until we sort some uid management stuff out)